altapay.php

```diff
@@ -275,8 +287,55 @@ function altapay_meta_box( $post ) {
  * @return void
  */
 function altapay_order_reconciliation_identifier_meta_box( $post ) {
-	// Fetch order reconciliation identifier
-	echo esc_html(get_post_meta( $post->ID, '_reconciliation_identifier', true ));
+
+	$settings = new Core\AltapaySettings();
+	$login    = $settings->altapayApiLogin();
+
+	if ( ! $login || is_wp_error( $login ) ) {
+		echo '<p>' . __( 'Could not connect to AltaPay!', 'altapay' ) . '</p>';
+		return;
+	}
+
+	$postID = $post->ID;
+
+	if ( $post->post_type === 'altapay_captures' ) {
+		$postID = wp_get_post_parent_id( $postID );
+	}
+
+	$auth = $settings->getAuth();
+
+	$order = new WC_Order( $postID );
+	$txnID = $order->get_transaction_id();
+
+	if ( $txnID ) {
+		$api = new Payments( $auth );
+		$api->setTransaction( $txnID );
+		$payments = $api->call();
+		?>
+		<table width="100%" cellspacing="0" cellpadding="10">
+		   <thead>
+		   <tr>
+			   <th align="left" width="40%">Reconciliation Identifier</th>
+			   <th align="left" width="60%">Type</th>
+		   </tr>
+		   </thead>
+			<tbody>
+				<?php
+				foreach ( $payments as $payment ) {
+					foreach ( $payment->ReconciliationIdentifiers as $identifier ) {
+						?>
+						<tr>
+							<td><?php echo $identifier->Id; ?></td>
+							<td><?php echo $identifier->Type; ?></td>
+						</tr>
```

Comment:
@emicha We are only showing the reconciliation identifiers using the `API/payments` endpoint. And we are not saving it with the orders on the CMS side. Also, WooCommerce doesn't have an option for export to CSV by default but some plugins are available to enable this feature. Can I create a ticket to save reconciliation identifiers with orders as well?

Reply:
@shahbaz-mehar Yes please do, otherwise the merchant needs to open manually each order, in order to be able to reconcile.
It is important that this ID is then populated in an export feature that the merchant might be using, or on a field that an ERP will use later on.

Reply:
Done